saksstatistikk_ef/src/main/kotlin/no/nav/familie/eksterne/kontrakter/saksstatistikk/Saksstatistikk.kt

```diff
@@ -0,0 +1,52 @@
+package no.nav.familie.eksterne.kontrakter.saksstatistikk
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import java.time.LocalDate
+import java.time.ZonedDateTime
+
+data class BehandlingDVH(
```

Comment:
Er dette det samme som https://github.com/navikt/familie-eksterne-kontrakter/blob/f81111ddd880c195a1610f600f958b87c6c5892b/saksstatistikk/src/main/kotlin/no/nav/familie/eksterne/kontrakter/saksstatistikk/Saksstatistikk.kt ? Trenger vi en for BA og en for EF?